Looks like even after the bump we're still failing the same test,

```
Jun 11 19:20:25 =================================== FAILURES ===================================
Jun 11 19:20:25 _____________ TestONNXRuntime_opset11.test_topk_smallest_unsorted ______________
Jun 11 19:20:25 
Jun 11 19:20:25 self = <test_pytorch_onnx_onnxruntime.TestONNXRuntime_opset11 testMethod=test_topk_smallest_unsorted>
Jun 11 19:20:25 
Jun 11 19:20:25     def wrapper(self):
Jun 11 19:20:25         if self.opset_version < min_opset_version:
Jun 11 19:20:25             raise unittest.SkipTest("Skip verify test for unsupported opset_version")
Jun 11 19:20:25 >       return func(self)
Jun 11 19:20:25 
Jun 11 19:20:25 test/onnx/test_pytorch_common.py:54: 
Jun 11 19:20:25 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
Jun 11 19:20:25 test/onnx/test_pytorch_onnx_onnxruntime.py:1491: in test_topk_smallest_unsorted
Jun 11 19:20:25     self.run_test(MyModule(), (x, k))
Jun 11 19:20:25 test/onnx/test_pytorch_onnx_onnxruntime.py:126: in run_test
Jun 11 19:20:25     _run_test(model)
Jun 11 19:20:25 test/onnx/test_pytorch_onnx_onnxruntime.py:122: in _run_test
Jun 11 19:20:25     fixed_batch_size=fixed_batch_size)
Jun 11 19:20:25 test/onnx/test_pytorch_onnx_onnxruntime.py:86: in run_model_test
Jun 11 19:20:25     ort_test_with_input(ort_sess, input_copy, output, rtol, atol)
Jun 11 19:20:25 test/onnx/test_pytorch_onnx_onnxruntime.py:47: in ort_test_with_input
Jun 11 19:20:25     [np.testing.assert_allclose(out, ort_out, rtol=rtol, atol=atol) for out, ort_out in zip(outputs, ort_outs)]
Jun 11 19:20:25 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
Jun 11 19:20:25 
Jun 11 19:20:25 .0 = <zip object at 0x7f9bb8e41888>
Jun 11 19:20:25 
Jun 11 19:20:25 >   [np.testing.assert_allclose(out, ort_out, rtol=rtol, atol=atol) for out, ort_out in zip(outputs, ort_outs)]
Jun 11 19:20:25 E   AssertionError: 
Jun 11 19:20:25 E   Not equal to tolerance rtol=0.001, atol=1e-07
Jun 11 19:20:25 E   
Jun 11 19:20:25 E   Mismatch: 66.7%
Jun 11 19:20:25 E   Max absolute difference: 2.
Jun 11 19:20:25 E   Max relative difference: 2.
Jun 11 19:20:25 E    x: array([1., 2., 3.], dtype=float32)
Jun 11 19:20:25 E    y: array([3., 2., 1.], dtype=float32)
Jun 11 19:20:25 
Jun 11 19:20:25 test/onnx/test_pytorch_onnx_onnxruntime.py:47: AssertionError

```

We're looking to close out the first RC for `1.5.1` today and this is our last cherry pick, is there any way we can look to get this resolved by EOD today @KsenijaS @spandantiwari?

--- a/.jenkins/caffe2/test.sh
+++ b/.jenkins/caffe2/test.sh
@@ -144,7 +144,7 @@ if [[ "$BUILD_ENVIRONMENT" == *onnx* ]]; then
     # default pip version is too old(9.0.2), unable to support tag `manylinux2010`.
     # Fix the pip error: Couldn't find a version that satisfies the requirement
     sudo pip install --upgrade pip
-    pip install -q --user -i https://test.pypi.org/simple/ ort-nightly==1.1.0.dev1228
+    pip install -q --user -i https://test.pypi.org/simple/ ort-nightly==1.3.0.dev202005123
   fi
   "$ROOT_DIR/scripts/onnx/test.sh"
 fi

--- a/test/onnx/test_pytorch_onnx_onnxruntime.py
+++ b/test/onnx/test_pytorch_onnx_onnxruntime.py
@@ -1484,7 +1484,11 @@ class TestONNXRuntime(unittest.TestCase):
     def test_topk_smallest_unsorted(self):
         class MyModule(torch.nn.Module):
             def forward(self, x, k):
-                return torch.topk(x, k, largest=False, sorted=False)
+                # When sorted=False, order of elements in the outout tensors
+                # are not expected to match between PyTorch and ORT
+                topk_unsorted = torch.topk(x, k, largest=False, sorted=False)
+                topk_sorted = torch.topk(x, k, largest=False, sorted=True)
+                return topk_sorted, torch.sort(topk_unsorted.values).values
 
         x = torch.arange(1., 6., requires_grad=True)
         k = torch.tensor(3)

